config/manager/manager.yaml

```diff
@@ -20,8 +20,9 @@ spec:
         - /manager
         args:
         - --enable-leader-election
-        image: controller:latest
+        image: goharbor/harbor-operator:dev
         name: manager
+        imagePullPolicy: IfNotPresent
```

Comment:
Not necessary, it's the default behavior for tags != 'latest'.
I think we should use default strategy.

Reply:
fixed